### PR TITLE
Move CLI page text into translation file

### PR DESCRIPTION
**Pull request by eignnx**
Work on internationalization re: #798 

Extracts all user-facing text (including img alt text) from the CLI page into a Fluent translation file.

### CODE REVIEW

File: templates/fluent-resource/en-US/cli.ftl [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+cli-pitch-ship-img-alt = box with a checkmark
+cli-pitch-ship-heading = Easy distribution
+cli-pitch-ship-description = Compile everything down to a single binary&mdash;no
+        need for your users to have a runtime or libraries installed.
+cli-pitch-ship-link = How to ship Rust code
+
+cli-pitch-config-img-alt = A note and a gear
+cli-pitch-config-heading = Robust configuration
+cli-pitch-config-description = Handle configuration files across platforms with
+        ease.  Rust will deal with namespaces and formats for you.
```
Comment (eignnx) [inline templates/fluent-resource/en-US/cli.ftl]: These 2 spaces before "Rust will deal..." were in the original markup.

File: templates/fluent-resource/en-US/cli.ftl [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+cli-pitch-config-link = Start configuring
+
+cli-pitch-manuals-img-alt = Help manual
+cli-pitch-manuals-heading = Manuals? done.
+cli-pitch-manuals-description = Generate manual pages for your app
+        automatically.  Just package the generated files and you’re done.
```
Comment (eignnx) [inline templates/fluent-resource/en-US/cli.ftl]: These 2 spaces before "Just package the..." were in the original markup.

**Comment by Manishearth** [comment]
Oops, I didn't notice this at all. Reviewing now

File: templates/fluent-resource/en-US/cli.ftl (hunk 1 above)
Comment (Manishearth) [reply templates/fluent-resource/en-US/cli.ftl]: It's HTML, it will be collapsed

**Comment by Manishearth** [comment]
Thanks!

**Pull request merged**